locale/it/translation.js

```diff
@@ -0,0 +1,415 @@
+    STAKE_NOT_SEND:
+        "In questo caso, utilizza la schermata <b>Stake</b>, NON la schermata Invia!", //Here, use the <b>Stake</b> screen, not the Send screen!
+    BAD_ADDR_LENGTH:
+        '<b>Indirizzo PIVX non valido!<b><br>Lunghezza errata ({addressLength})', //<b>Invalid PIVX address!<b><br>Bad length ({addressLength})
+    BAD_ADDR_PREFIX:
+        '<b>Indirizzo PIVX non valido!<b><br>Prefisso {address} errato (dovrebbe iniziare con {addressPrefix})', //<b>Invalid PIVX address!<b><br>Bad prefix {address} (Should start with {addressPrefix})
+    SENT_NOTHING: "Non puoi inviare \'niente\'!", //You can\'t send \'nothing\'!
+    MORE_THEN_8_DECIMALS: 'limite di 8 decimali superato', //8 decimal limit exceeded
+    SAVE_WALLET_PLEASE:
+        '<b>Salva il tuo portafoglio!</b><br>Dashboard ➜ Proteggi il tuo portafoglio', //<b>Save your wallet!</b><br>Dashboard ➜ Secure your wallet
+    BACKUP_OR_ENCRYPT_WALLET:
+        'Ti preghiamo di crittografare e/o eseguire il backup delle tue chiavi prima di partire, altrimenti potresti perderle!', //Please ENCRYPT and/or BACKUP your keys before leaving, or you may lose them!
+    NO_CAMERAS: "Questo dispositivo non ha una fotocamera!", //This device has no camera!
+    STAKING_LEDGER_NO_SUPPORT: "Il Ledger non è supportato per il Cold Staking", //Ledger is not supported for Cold Staking
+    CONNECTION_FAILED:
+        '<b>Impossibile sincronizzare!</b> Riprova più tardi.<br>Puoi provare a riconnetterti tramite le Impostazioni.', //<b>Failed to synchronize!</b> Please try again later.<br>You can attempt re-connect via the Settings.
+    MN_NOT_ENABLED: "Il tuo masternode non è ancora abilitato!", //Your masternode is not enabled yet!
+    VOTE_SUBMITTED: 'Voto inviato!', //Vote submitted!
+    VOTED_ALREADY:
+        'Hai già votato per questa proposta! Si prega di attendere 1 ora', //You already voted for this proposal! Please wait 1 hour
+    VOTE_SIG_BAD:
+        'Impossibile verificare la firma, controlla la chiave privata del tuo masternode', //Failed to verify signature, please check your masternode\'s private key
+    MN_CREATED_WAIT_CONFS:
+        '<b>Masternode creato!<b><br>Attendere 15 conferme per procedere ulteriormente', //<b>Masternode Created!<b><br>Wait 15 confirmations to proceed further
+    MN_ACCESS_BEFORE_VOTE: 'Accedi a un masternode prima di votare!', //Access a masternode before voting!
+    MN_OFFLINE_STARTING:
+        'Il tuo masternode è offline, proviamo ad avviarlo', //Your masternode is offline, we will try to start it
+    MN_STARTED: '<b> Il Masternode è avviato!</b>', //<b>Masternode started!</b>
+    MN_RESTARTED: '<b>Il Masternode è stato riavviato<b/>', //<b>Masternode restarted!</b>
+    MN_STARTED_ONLINE_SOON:
+        '<b>Masternode è iniziato!</b><br>Sarà presto online', //<b>Masternode started!</b><br>It\'ll be online soon
+    MN_START_FAILED: '<b>Masternode è iniziato!</b>', //<b>Masternode started!</b>
```

Comment:
MN_START_FAILED is supposed to be "Masternode couldn't be started", this seems to be an error in the english translation too